ChangeLog

```diff
@@ -16,6 +16,8 @@
  * EnclosingSimplex, EnclosingSimplexImplementation, NaiveEnclosingSimplex, BoundingVolumeHierarchy
  * IndicesCollection
  * SymbolicParserExprTk
+ * EvaluationProxy
+ * MemoizeFunction, MemoizeEvaluation
```

Comment:
The documentation should be updated.

Reply:
Grrrr I always forget to add new classes to rst, thanks.